packages/react-focus/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/react-focus",
+  "version": "0.0.1",
```

Comment:
Is this intentional? create-package usually does 0.1.0 to avoid the weirdness of 0.0.x semver.

Reply:
i wasn't sure what the weirdness was. I can certainly revert but what's the weirdness?

Reply:
I think it's mainly that `^0.0.1` will only pick up **specifically** that version (essentially `^` or `~` is ignored).

Also IIRC `^` behaves like `~` for `0.(>0).x`, but that's not quite as surprising.